data/sql/updates/pending_db_world/rev_1588339146103973500.sql

```diff
@@ -0,0 +1,36 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1588339146103973500');
+
+/*
+ * Zone: Sholazar Basin
+ * Update by Knindza | <www.azerothcore.org>
+ * Copyright (C) <www.shadowburn.net> & <www.lichbane.com>
+*/
+
+/* FLAGS */
+UPDATE `creature_template` SET `unit_flags` = 4 WHERE `entry` = 29125;
+UPDATE `creature_template` SET `unit_flags` = 4 WHERE `entry` = 29126;
+UPDATE `creature_template` SET `unit_flags` = 4 WHERE `entry` = 29127;
```

Comment:
```suggestion
UPDATE `creature_template` SET `unit_flags` = `unit_flags`|4 WHERE `entry` = 29125;
UPDATE `creature_template` SET `unit_flags` = `unit_flags`|4 WHERE `entry` = 29126;
UPDATE `creature_template` SET `unit_flags` = `unit_flags`|4 WHERE `entry` = 29127;
```

Reply:
Generally, when working with flags we don't want to override possible flags, but we want to add them ontop on existing ones with | (this case it doesn't really matter since there are none, but it is just a general guideline :D)